packages/stark-core/package.json

```diff
@@ -64,17 +56,18 @@
     "@angular/compiler-cli": "5.x",
```

Comment:
Are we sure we need the angular/compiler-cli in Stark Core? and angular/compiler as well?

Reply:
I'm not sure it's needed. Currently, they are part of the stark-build and stark-core.

Reply:
To me, we should remove them unless they are really needed. So if you remove them and everything works just fine...

Reply:
For sure, because the deps will be available in the root :)
But ok, I'll remove those peerDeps from stark-core